evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -42,16 +40,15 @@ def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, ma
                       "max_depth": max_depth,
                       "num_leaves": num_leaves,
                       "min_child_samples": min_child_samples,
-                      "n_jobs": n_jobs,
-                      "bagging_freq": bagging_freq,
-                      "bagging_fraction": bagging_fraction}
+                      "n_jobs": n_jobs}
         parameters.update(kwargs)
         lg_parameters = copy.copy(parameters)
         # when boosting type is random forest (rf), LightGBM requires bagging_freq == 1 and  0 < bagging_fraction < 1.0
         if boosting_type == "rf":
             lg_parameters['bagging_freq'] = 1
-        # avoid lightgbm warnings having to do with parameter aliases
-        if lg_parameters['bagging_freq']:
+            if 'bagging_fraction' not in lg_parameters.keys():
+                lg_parameters['bagging_fraction'] = 0.9
+            # avoid lightgbm warnings having to do with parameter aliases
```

Comment:
We only add bagging if `boosting type=rf`, and then we set the `subsample` args to None to remove the warnings in AutoMLSearch.